src/iosource/pcap/pcap.bif

```diff
@@ -102,3 +106,54 @@ function error%(%): string
 
 	return zeek::make_intrusive<zeek::StringVal>("no error");
 	%}
+
+function findalldevs%(%): Pcap::Interfaces
+	%{
+	pcap_if_t *alldevs, *d;
+	char errbuf[PCAP_ERRBUF_SIZE];
+
+	int ret = pcap_findalldevs(&alldevs, errbuf);
```

Comment:
Ugh, my bad.  I started to add this yesterday (and I learned about the emit_builtin_error call), but then I backed it out because the man page said that a '0' ret value is not considered a failure.  Now I see that it can also return PCAP_ERROR.  

Will fix!